src/send_patch.sh

```diff
@@ -6,6 +6,7 @@
 include "${KW_LIB_DIR}/lib/kw_config_loader.sh"
 include "${KW_LIB_DIR}/lib/kwlib.sh"
 include "${KW_LIB_DIR}/lib/kw_string.sh"
+include "./src/kw_manager.sh"
```

Comment:
Can we handle this commit after we figure out the contact part? It could be a new PR. Right now, let's just focus on the group.